accounts/auth.go

```diff
@@ -0,0 +1,63 @@
+package accounts
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/user"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+type Token struct {
+	token []byte
+}
+
+func Hostname() string {
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Finding Hostname", err)
+	}
+	return hostname
+}
```

Comment:
I'm not a big fan of panic'ing on every error - this will start to be very hard to keep track of (in my opinion, it already is). I see that you're doing this across the entire project for now, so I'm not commenting on every single occurrence.

Reply:
No, it's pretty bad. It's an old habit that when I don't want to focus on it, just go fatal and stop. Works for a first pass and then I can grep Fatal later and fix it properly.

Reply:
That's fine. I think I left a similar comment on this PR already, but it's sometimes hard to grasp the intention of committed code. When in doubt, just add some `TODO` comments explaining any shortcuts.

Reply:
I've worked with different teams over the years that have essentially spanned the spectrum of error philosophies, my favorite is to stop immediately in dev, but muddle through in prod. I've written about it a few times: http://theprogrammersparadox.blogspot.com/2014/11/error-handling.html?m=0